pkg/common/utils/resource/pod.go

```diff
@@ -85,6 +85,18 @@ func NewPodTemplateSpec(dcr *v1.DorisCluster, componentType v1.ComponentType) co
 			Affinity:           spec.Affinity,
 			Tolerations:        spec.Tolerations,
 			HostAliases:        spec.HostAliases,
+			InitContainers: []corev1.Container{
```

Comment:
it is not for all components, distinct different components to initial.
now, only `be` needs an initial in the host.